chia/full_node/fee_estimate.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+from chia.types.fee_rate import FeeRate
+from chia.util.ints import uint64
+from chia.util.streamable import Streamable, streamable
+
```

Comment:
Excellent question. This one I'm going to write up as documentation, because it will be pretty long.

I'll comment here and resolve this when I point you to the writeup. Thank you.

Reply:
By the way @almogdepaz the error case from this Estimator is at line 81 in chia/full_node/fee_estimator.py

https://github.com/Chia-Network/chia-blockchain/pull/13479/files#diff-045b40539e9c7cea0d135fcd57dc01809d8d5fa06978feb2f9cc2c799e56d95cR81

Reply:
It's the `-1` case in the Bitcoin source code.